docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -138,7 +138,11 @@ The configuration above generates the following sidebar:
 
 Starlight can automatically generate a group in your sidebar based on a directory of your docs.
 This is helpful when you do not want to manually enter each sidebar item in a group.
-Pages will be sorted alphabetically by filename by default.
+By default, pages will be sorted in alphabetical order according to the filenames in your [`defaultLocale`](/reference/configuration/#defaultlocale). 
```

Comment:
```suggestion

By default, pages will be sorted in alphabetical order according to the filenames in your [`defaultLocale`](/reference/configuration/#defaultlocale). 
```
I will let you decide, or some 3rd-party tie breaker, but I *do* prefer this as a separate line. I don't like burying key information like "what the default is!" at the end of an already long-ish paragraph. And, I think this sets up the tip nicely. There's no way someone is not going to take away "alphabetical" after seeing all this, AND FURTHERMORE that "alphabetical" might not even be what they expect.

So, non-blocking, and I'll allow merging either way, but this is def my personal preference.